### PR TITLE
Cleaning up some old school string concatenation, this is the new school, we use ES6 template strings.

### PR DESCRIPTION
**Pull request by zanthrash**

### CODE REVIEW

**Comment by anotherchrisberry** [comment]
Old school concatenation was consciously chosen here because of performance concerns on huge applications. Can you check the performance against some of the huge applications at Netflix to see how they handle using template strings?

**Comment by zanthrash** [comment]
Fair point.

**Comment by zanthrash** [comment]
Actually they get transpiled to concated  strings in Chrome and FF.  Because we are running them thru Babel we are not using the native browser support for string templates.

File: app/scripts/modules/core/instance/instanceListBody.directive.js [hunk 1]
```diff
@@ -145,6 +137,7 @@ module.exports = angular.module('spinnaker.core.instance.instanceListBody.direct
         }
 
         function renderInstances() {
+          console.log('renderInstances');
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/instance/instanceListBody.directive.js]: remove

**Comment by anotherchrisberry** [comment]
Did some testing locally and it doesn't affect performance so I'm good with this.

File: app/scripts/modules/core/instance/instanceListBody.directive.js [hunk 2]
```diff
@@ -5,8 +5,9 @@ let angular = require('angular');
 module.exports = angular.module('spinnaker.core.instance.instanceListBody.directive', [
   require('../cluster/filter/clusterFilter.service.js'),
   require('../cluster/filter/clusterFilter.model.js'),
+  require('../../../modules/core/utils/rx'),
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/instance/instanceListBody.directive.js]: doesn't look like you're using this

**Pull request merged**